cli/command/context/remove.go

```diff
@@ -26,7 +28,7 @@ func newRemoveCommand(dockerCli command.Cli) *cobra.Command {
 			return RunRemove(dockerCli, opts, args)
 		},
 	}
-	cmd.Flags().BoolVarP(&opts.Force, "force", "f", false, "Force the removal of a context in use")
```

Comment:
I'll temporarily change this back to the old string; while it's not "fully" accurate, the docs need some improvements as well for this (no description for this flag), so I'll update the flag description together with improving the docs in a follow-up